packages/vrm-viewer/tsconfig.json

```diff
@@ -0,0 +1,8 @@
+{
+  "extends": "../../tsconfig.json",
+  "include": ["./src"],
+  "compilerOptions": {
+    "module": "ESNext",
+    "experimentalDecorators": true
```

Comment:
What is this `experimentalDecorators`? is this required to compile the project with the `@google/model-viewer` dependency?

Reply:
Not used directly for now but required for building types for `@google/model-viewer`.   It's used for [extending web components's properties]( https://github.com/google/model-viewer/blob/master/packages/model-viewer/src/features/animation.ts#L52-L56).  We may want to add `vrma="..."` via `@property` too later.